internal/kibana/reconciler.go

```diff
@@ -482,6 +501,16 @@ func findImageStreamTag(tags []imagev1.NamedTagEventList, name string) *imagev1.
 	return nil
 }
 
+func findTagReferencePolicy(tags []imagev1.TagReference, name string) *imagev1.TagReferencePolicy {
```

Comment:
If you directly return the `ReferencePolicy.Type` instead of a pointer to `ReferencePolicy` you can save yourself one nil-check in the calling function.

Reply:
the nil pointer is intentional as it indicate that no referencePolicy was found

Reply:
There are no pointers in the original type, so you can be sure that there is no nil-references in there and use `""` as your "nothing is set" fallback.

Reply:
sure why not